src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -170,7 +178,8 @@ const SignOrExecuteForm = ({
 
   const cannotPropose = !isOwner && !onlyExecute // Can't sign or create a tx if not an owner
   const submitDisabled = !isSubmittable || isEstimating || !tx || disableSubmit || isWrongChain || cannotPropose
-  const error = props.error || (willExecute && gasLimitError)
+  const error = props.error || (willExecute ? gasLimitError || executionValidationError : undefined)
+  const isSendingToSelf = wallet ? sameString(wallet?.address, safeAddress) : false
```

Comment:
A misnomer, it's not sending to self, it's the Safe executing its own tx.
Sending to self works fine.